docs/index.md

```diff
@@ -0,0 +1,58 @@
+# {Charm name} Operator
+
+<!-- A single sentence that says what the product is, succinctly and memorably. Add a 1-2 sentence description of what the charm software does. -->
+
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing {Charm software} on 
```

Comment:
Perhaps use `<...>` brackets to indicate parts that should be replaced, that is commonly used in dev docs

Reply:
Done, thanks for the suggestion!